### ISSUE TITLE
Multiple Monero hosts 

### ISSUE THREAD

**Issue by SnAFKe** [SChernykh/p2pool#269]
Hello, i've a question with this.

- Support multi ports too, some host no use same port for example if you use three host and there host use different port is possible to connect.
- Possible Failover? what happen if you have 5 host and then three is down and later "main host" is back, it's connect again with the main or still using current host.
- Also what happen if host use rpc-login? and multi host use different logins.

**Comment by SChernykh** [comment]
- You can read more in documentation: https://github.com/SChernykh/p2pool/blob/master/docs/COMMAND_LINE.MD#multiple-backup-hosts
- P2Pool does cycle through hosts until it finds a working one.
- rpc-login has to be specified for each host in the command line, if this host uses it

**Comment by SnAFKe** [comment]
> * You can read more in documentation: https://github.com/SChernykh/p2pool/blob/master/docs/COMMAND_LINE.MD#multiple-backup-hosts

Let me rephrase again, if you have four hosts and these four hosts use different ports just specify?

```host1 port:1111 host2 port:2222 host3 port:3333 host4 port:4444```
